src/inputCL.cpp

```diff
@@ -116,6 +116,8 @@ void InputCL::read_args( int argc, char *argv[] )
                  "                                (N = positive integer)\n",
                  "                                (PATH = location of ",
                  "directory)\n" );
+            log( std::cout, "  --sparse : "
```

Comment:
I would prefer `--vacuum`, but not a big deal either way

Reply:
Renamed to `--vacuum`.